mlos_bench/mlos_bench/config/schemas/environments/common-environment-subschemas.json

```
@@ -79,6 +79,10 @@
                 "teardown": {
                     "description": "The command lines to execute for the teardown phase.",
                     "$ref": "#/$defs/command_lines"
+                },
+                "parse_results_stdout": {
```

```suggestion
                "results_stdout_regex": {
```

or 

```suggestion
                "results_stdout_pattern": {
```

Minor naming suggestion to help understand what the item is quickly.

---

I agree. I like the latter more